appdata.patch

```diff
@@ -0,0 +1,8 @@
+--- a/org.kde.lokalize.appdata.xml	2020-06-09 01:03:51.000000000 +0700
++++ b/org.kde.lokalize.appdata.xml	2020-06-20 21:45:11.981635367 +0700
+@@ -139,4 +139,5 @@
+     <release version="20.04.0" date="2020-04-23"/>
+     <release version="19.12.1" date="2020-01-09"/>
+   </releases>
++  <content_rating type="oars-1.1" />
```

Comment:
Could you submit a Merge Request upstream about this?

https://invent.kde.org/sdk/lokalize

Reply:
https://invent.kde.org/sdk/lokalize/-/merge_requests/1

Reply:
Ideally you would have made that against release/20.04 so it gets included in 20.04.3 released next month, but that's fine, i guess we can wait carrying the patch until 20.08